### PR TITLE
Test add new blocks

### PR DESCRIPTION
**Pull request by CAmi307**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.

### Description of the Change

New blocks dedicated to Raspberry Pi Pico board have been added, allowing interaction between the board and some peripherals like thermistors, servo motors or distance sensors. This PR also contains some new blocks dedicated to Wifi connection and to communication through Wifi.

### Format

[x] ran `npm run electron-format`
[x] ran `npm run browser-format`

### Author
Signed-off-by: Amalia Camelia Simion <amalia.simion@wyliodrin.com>

### CODE REVIEW

File: source/plugins/projects/editor.visual/visual/language_definitions.js [hunk 1]
```diff
@@ -84,27 +84,6 @@ module.exports = function (blockly) {
...
-			this.appendStatementInput('NAME')
-				.setCheck('null')
-				.appendField('do');
-			this.setInputsInline(true);
-			this.setPreviousStatement(true);
-			this.setNextStatement(true);
-			this.setTooltip('');
-		}
-	};
-
```
Comment (alexandruradovici) [inline source/plugins/projects/editor.visual/visual/language_definitions.js:107]: this should not be deleted

File: source/plugins/projects/editor.visual/visual/language_python.js [hunk 2]
```diff
@@ -111,31 +113,31 @@ module.exports = function (blockly) {
...
-		for (var i = 0; i < globals.length; i++) {
-			globals[i] = Blockly.Python.variableDB_.getName(globals[i], Blockly.Variables.NAME_TYPE);
-		}
-		globals = globals.length ? '  global ' + globals.join(', ') + '\n' : '';
-		// TODO: Assemble Python into code variable.
-		var code = 'def ' + dfunct + '():\n' + globals + statements_name +
-			'  Timer(' + value_value + ', ' + dfunct + ').start()\n' +
-			dfunct + '()\n';
-		return code;
-	};
```
Comment (alexandruradovici) [inline source/plugins/projects/editor.visual/visual/language_python.js:138]: This should not be deleted.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: source/plugins/projects/editor.visual/visual/language_python.js (hunk 2 above)
Comment (CAmi307) [reply source/plugins/projects/editor.visual/visual/language_python.js]: added it back

File: source/plugins/projects/editor.visual/visual/language_definitions.js (hunk 1 above)
Comment (CAmi307) [reply source/plugins/projects/editor.visual/visual/language_definitions.js]: added it back

**Review by alexandruradovici — APPROVED** [review]

**Pull request merged**